fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflow.java

```diff
@@ -44,6 +47,12 @@
 @Singleton
 public class DruidWorkflow implements RequestWorkflowProvider {
 
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+    private int druidUncoveredIntervalLimit = SYSTEM_CONFIG.getIntProperty(
```

Comment:
@garyluoex I can make it final. The only problem is that groovy tests won't be able to pass on a this final variable test https://github.com/yahoo/fili/blob/a750c180e29b9455139be0bde18031ea232ee8ad/fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/workflow/DruidWorkflowSpec.groovy#L211